src/encoded/static/scss/encoded/modules/_panels.scss

```diff
@@ -726,6 +731,11 @@ $file-gallery-controls-height: 34px; // Height of controls inside file-gallery-c
     }
 }
 
+.collapsing-title-file-count {
+    font-size: 0.7em;
```

Comment:
Did you mean `rem`? We haven’t used ems because of their cascading problem.

Reply:
Done.